lms/envs/production.py

```diff
@@ -1068,3 +1068,6 @@ def get_env_setting(setting):
 
 ############## DRF overrides ##############
 REST_FRAMEWORK.update(ENV_TOKENS.get('REST_FRAMEWORK', {}))
+
+################################# CELERY ######################################
+CELERY_IMPORTS = tuple({*CELERY_IMPORTS, *ENV_TOKENS.get('CELERY_IMPORTS', ())})
```

Comment:
I may be missing something, because I didn't read through everything, but would it make sense to make an env variable with a different name, like `CELERY_IMPORTS_EXTENDED`?

Reply:
Something like this?
```suggestion
CELERY_IMPORTS = tuple({*CELERY_IMPORTS, *ENV_TOKENS.get('CELERY_IMPORTS_EXTENDED', ())})
```
I can change it :D

Reply:
Thanks. Before updating, I just want to check on the naming convention. See https://discuss.openedx.org/t/setting-naming-convention-question-for-extending-list-or-tuple/6967

Reply:
@MaferMazu: What do you think of the comments on the post (see post for details):
* `CELERY_EXTRA_IMPORTS` (as new name)
* Proposals for switching from tuple to list.

Reply:
@MaferMazu: It is unclear if you are in the forums. There is other useful discussion in that thread, but just in case you aren't, I will post one response.

From @ormsbee:
> I’m not quite sure if I understand what’s being asked, so I’m going to re-state the issue (and please correct me if I get this wrong).
>
> 1. Celery requires a setting called `CELERY_IMPORTS` that can be any [sequence of strings](http://www.pythondoc.com/celery-3.1.11/configuration.html#celery-imports) that represent modules to import when the worker starts.
> 2. We define a setting called `CELERY_IMPORTS` as a tuple, and put some modules that must be there in order for the system to work.
> 3. We want to define an environment variable in the YAML file that makes it clear that this value is not going to replace `CELERY_IMPORTS`, but will supplement it with additional modules that should be added for this particular installation. This will be named `CELERY_IMPORTS_EXTENDED`.
> 
> Code snippet:
> 
> `CELERY_IMPORTS = tuple({*CELERY_IMPORTS, *ENV_TOKENS.get('CELERY_IMPORTS_EXTENDED', ())})`
>
> We do use the convention of `EXTRA_` in a small number of cases, like `EXTRA_MIDDLEWARE_CLASSES` or `XBLOCK_EXTRA_MIXINS`. One small suggestion I would make is to make `CELERY_IMPORTS` a list and not a tuple, so that the code can read like:
> 
> `CELERY_IMPORTS.extend(ENV_TOKENS.get('CELERY_IMPORTS_EXTENDED', []))`
>
> Another possibility would be to do something like:
>
> `CELERY_IMPORTS += ENV_TOKENS.get('CELERY_IMPORTS_EXTENDED`, (,))`
>
> We probably don’t want to splat it into an intermediate set though, because we would lose ordering guarantees, which might bite us in weird and unexpected ways in the future.

Reply:
Thanks @robrap I already read the discuss 🙌